exercise/function/shapes.c

```diff
@@ -0,0 +1,28 @@
+/*Calculate the circumference and area of the circle*/
+
+#include <stdio.h>
+float circum(float r1);
+float area(float r2);
+#define Pi 3.14
+
+int main(void)
+{
+    printf("enter radius: ");
+    float r1, r2;
```

Comment:
- [ ] Should be follow rules:
1. Always definition/declaration variables first
2. Add empty line
3. Start first statement: printf("Enter radius: ")